pyuvdata/uvbeam.py

```diff
@@ -143,11 +143,16 @@ def __init__(self):
                                           expected_type=int)
 
         desc = ('Normalization standard of data_array, options are: '
-                '"peak" and "solid_angle". Peak normalized means each frequency '
-                'is separately normalized such that the peak is 1 and the beam '
-                'is dimensionless. Solid angle normalized means the peak '
-                'normalized beam is divided by the integral of the beam '
-                'over the sphere, giving the beam dimensions of 1/stradian.')
+                '"physical", "peak" or "solid_angle". Physical normalization '
+                'means that the frequency dependence of the antenna sensitivity '
+                'is included in the data_array while the frequency dependence '
+                'of the receiving chain is included in the bandpass_array. '
+                'Peak normalized means that for each frequency the data_array'
+                'is separately normalized such that the peak is 1 (so the beam '
+                'is dimensionless) and all frequency dependence is moved to the '
+                'bandpass_array. Solid angle normalized means the peak normalized '
+                'beam is divided by the integral of the beam over the sphere, '
```

Comment:
It depends on the type. I'll clarify a bit, it really makes the most sense for the power beam, but it can be defined for voltage beam as well.